profiles/common/modules/custom/tmgmt_workbench/tmgmt_workbench.module

```diff
@@ -114,55 +113,62 @@ function tmgmt_workbench_form_tmgmt_entity_ui_translate_form_alter(&$form, $form
       $state = $node->workbench_moderation['current']->state;
       $translation_possible = in_array($state, $states, 'strict');
 
-      $languages = tmgmt_available_languages();
-      $options = &$form['languages']['#options'];
+      $options = $form['languages']['#options'];
+
+      $existing_jobs = tmgmt_job_item_load_latest(
+        'workbench_moderation',
+        $entity_type,
+        $node->nid,
+        $node->translations->original
```

Comment:
Poetry only send fields that are marked as translatable with entity translation. If you try to send something using i18n i guess the submission will be empty. I guess this could be added to the documentation's remark section later on. I'll create a ticket.